pegasus/helpers/form_helpers.rb

```diff
@@ -93,6 +89,7 @@ def update_form(kind, secret, data)
 
   prev_data = JSON.parse(form[:data], symbolize_names: true)
   symbolized_data = Hash[data.map {|k, v| [k.to_sym, v]}]
+  symbolized_data.merge!(Hash[JSON.parse(symbolized_data[:data]).map {|k, v| [k.to_sym, v]}])
```

Comment:
Agreed, I should have changed existing bad naming patterns. Let me do so...

As for `deep_symbolize_keys!`, much more readable.